.github/workflows/validate-inventory.yml

```diff
@@ -0,0 +1,21 @@
+name: Validate Inventory
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install SR tools
+      run: |
+        python -m pip install --upgrade pip
+        pip install sr.tools
```

Comment:
Is it possible to cache this install? I'm not familiar enough with GHA to know what runtime it uses.

Reply:
You cannot, which is unfortunate. It's possible it'll be added in a future version, but the build is only ~30s, which is fine in this situation.

Reply:
Support for caching has been [discussed](https://github.com/actions/toolkit/issues/47#issuecomment-521108865) and is apparently "on the feature backlog short list".

Reply:
https://github.com/actions/cache seems to have since appeared.